### PR TITLE
Zigbee fingerbot plus

### PR DESCRIPTION
**Pull request by kirovilya**
https://github.com/Koenkk/zigbee2mqtt/issues/17841
https://github.com/Koenkk/zigbee2mqtt/issues/18011

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -4903,6 +4902,57 @@ const definitions: Definition[] = [
...
+        description: 'Zigbee fingbot plus',
+        whiteLabel: [
+            tuya.whitelabel('Adaprox', 'TS0001_1', 'Zigbee fingbot plus', ['_TZ3210_dse8ogfy']),
+        ],
+        fromZigbee: [fz.on_off, tuya.fz.datapoints],
+        toZigbee: [tz.on_off,
+            {
+                ...tuya.tz.datapoints,
+                key: tuya.tz.datapoints.key.filter((item) => {
+                    return item !== 'state';
```
Comment (Koenkk) [inline src/devices/tuya.ts:4918]: Since tz.on_off is listed first this shouldnt be needed (?)

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -4903,6 +4902,57 @@ const definitions: Definition[] = [
             tuya.whitelabel('Marmitek', 'SM0202_1', 'Motion sensor', ['_TYZB01_yr95mpib']),
         ],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0001', ['_TZ3210_dse8ogfy']),
+        model: 'FingerBot',
+        vendor: 'TuYa',
+        description: 'Zigbee fingbot plus',
```
Comment (screamy) [inline src/devices/tuya.ts:4909]:
```suggestion
        description: 'Zigbee fingerbot plus',
```

File: src/devices/tuya.ts (hunk 1 above)
Comment (kirovilya) [reply src/devices/tuya.ts]: fixed

**Comment by kirovilya** [comment]
can merge

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**